scripts/cmake/utilities.cmake

```diff
@@ -19,3 +22,438 @@ function(mz_target_handle_warnings MZ_TARGET)
+#     [MACOS_SOURCES <macos_sources>]
+#     [LINUX_SOURCES <linux_sources>]
+#     [WINDOWS_SOURCES <windows_sources>]
+#     [WASM_SOURCES <wasm_sources>]
+#     [DUMMY_SOURCES <dummy_sources>]
+#     [TEST_SOURCES <test_sources>]
+#     [QT_DEPENDENCIES <qt_dependencies>]
+#     [MZ_DEPENDENCIES <mz_dependencies>]
+#     [RUST_DEPENDENCIES <rust_dependencies>]
+#     [EXTRA_DEPENDENCIES <extra_dependencies>]
+#     [TEST_DEPENDENCIES <test_dependencies>]
+#     [ANDROID_DEPENDENCIES <android_dependencies>]
+#     [MACOS_DEPENDENCIES <macos_dependencies>]
+#     [LINUX_DEPENDENCIES <linux_dependencies>]
+#     [WINDOWS_DEPENDENCIES <windows_dependencies>]
+#     [WASM_DEPENDENCIES <wasm_dependencies>]
+#     [DUMMY_DEPENDENCIES <dummy_dependencies>]
+# )
+#
+# Parameters:
+# - TARGET_NAME: The name of the target module.
+# - INCLUDE_DIRECTORIES: (Optional) List of additional include directories for the module.
+# - SOURCES: (Optional) List of source files for the module.
+# - IOS_SOURCES: (Optional) List of iOS-specific source files for the module.
+# - ANDROID_SOURCES: (Optional) List of Android-specific source files for the module.
+# - MACOS_SOURCES: (Optional) List of macOS-specific source files for the module.
+# - LINUX_SOURCES: (Optional) List of Linux-specific source files for the module.
+# - WINDOWS_SOURCES: (Optional) List of Windows-specific source files for the module.
+# - WASM_SOURCES: (Optional) List of WebAssembly-specific source files for the module.
+# - DUMMY_SOURCES: (Optional) List of dummy sources for the module.
+# - TEST_SOURCES: (Optional) List of test source files for the module.
+# - QT_DEPENDENCIES: (Optional) List of Qt dependencies for the module.
+# - MZ_DEPENDENCIES: (Optional) List of custom module dependencies for the module.
+# - RUST_DEPENDENCIES: (Optional) List of Rust dependencies for the module.
+# - EXTRA_DEPENDENCIES: (Optional) List of additional dependencies for the module.
+# - TEST_DEPENDENCIES: (Optional) List of test-only dependencies.
+#   If the name of the dependency starts with `replace-<targetname>`
+#   it will replace <targetname> dependency for tests.
+# - IOS_DEPENDENCIES: (Optional) List of iOS-specific dependencies for the module.
+# - ANDROID_DEPENDENCIES: (Optional) List of Android-specific dependencies for the module.
+# - MACOS_DEPENDENCIES: (Optional) List of macOS-specific dependencies for the module.
+# - LINUX_DEPENDENCIES: (Optional) List of Linux-specific dependencies for the module.
+# - WINDOWS_DEPENDENCIES: (Optional) List of Windows-specific dependencies for the module.
+# - WASM_DEPENDENCIES: (Optional) List of WebAssembly-specific dependencies for the module.
+# - DUMMY_DEPENDENCIES: (Optional) List of dummy dependencies for the module.
+#
+# Example:
+# MZ_ADD_NEW_MODULE(
+#     TARGET_NAME MyModule
+#     INCLUDE_DIRECTORIES ${CMAKE_CURRENT_SOURCE_DIR}/include
+#     GENERATED_SOURCES ${CMAKE_CURRENT_BINARY_DIR}/generated_sources.cpp
```

Comment:
Oh, this is wrong. And I need to send a new commit...

Reply:
https://github.com/mozilla-mobile/mozilla-vpn-client/pull/9090/commits/ac4f63f29b656f75f45247cc41dad977e4cc8093